### PR TITLE
Allow to edit chunter messages/comments

### PR DESCRIPTION
**Pull request by haiodo**
Allow to edit chunter messages/comments

Closes: #398

Signed-off-by: Andrey Sobolev <haiodo@gmail.com>

### CODE REVIEW

File: plugins/chunter-impl/src/components/Message.svelte [hunk 1]
```diff
@@ -135,43 +156,103 @@
...
-          <span>
-            <DateTime value={message.modifiedOn} timeOnly={isToday(message.modifiedOn)} />
-          </span>
-        </div>
-        {#if !thread}
+      {#if !editMode}
+        <div class="header">
+          <div>
+            {#if user}
+              {user?.name ?? ''}
```
Comment (BykhovDenis) [inline plugins/chunter-impl/src/components/Message.svelte:163]: Why we need `user?.` in `#if user`

File: plugins/chunter-impl/src/components/Message.svelte [hunk 2]
```diff
@@ -53,6 +62,11 @@
 
   const client = getClient()
 
+  $: currentId = client.accountId()
```
Comment (BykhovDenis) [inline plugins/chunter-impl/src/components/Message.svelte:65]: Why not const?

File: plugins/chunter-impl/src/components/Message.svelte (hunk 1 above)
Comment (haiodo) [reply plugins/chunter-impl/src/components/Message.svelte:163]: Not required. Thanks.

File: plugins/chunter-impl/src/components/Message.svelte (hunk 2 above)
Comment (haiodo) [reply plugins/chunter-impl/src/components/Message.svelte:65]: Fixed.

**Review by BykhovDenis — APPROVED** [review]

**Pull request merged**